src/Workspaces/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -1,3 +1,4 @@
+Microsoft.CodeAnalysis.Solution.WithProjectDocumentsOrder(Microsoft.CodeAnalysis.ProjectId projectId, System.Collections.Immutable.ImmutableList<Microsoft.CodeAnalysis.DocumentId> documentIds) -> Microsoft.CodeAnalysis.Solution
```

Comment:
@jasonmalinowski seems I can compile now. This may have been in a bug when I started this PR with dev16-vs-deps. But now in master-vs-deps it appears to be fine.